thirdeye/thirdeye-frontend/app/pods/manage/alerts/template.hbs

```diff
@@ -0,0 +1,144 @@
+<aside class="manage-alert-filters col-md-3">
```

Comment:
Actually, I tested this and it seems that the .row class is the one that adds -15px (which is not needed).
<img width="328" alt="screen shot 2017-06-23 at 2 32 35 pm" src="https://user-images.githubusercontent.com/11814420/27501209-cee8acd0-5820-11e7-9dbf-2cb64acf5b0a.png">